projects/plugins/jetpack/readme.txt

```diff
@@ -293,40 +293,104 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
 
 
 == Changelog ==
-### 12.3-a.7 - 2023-06-19
+### 12.3-beta - 2023-06-26
 #### Enhancements
 - AI Assistant: add and use ImproveToolbarDropdownMenu in block toolbar.
+- AI Assistant: add a specific delimiter for content in the prompts.
```

Comment:
I don't know if we need all those entries in the final readme.txt; that's a bit overwhelming. I would rather see groups of related changes that are easier to parse for users.

Reply:
https://github.com/Automattic/jetpack/pull/31600#issuecomment-1609690430